.github/workflows/test-jinad.yml

```diff
@@ -0,0 +1,56 @@
+name: Test jinad
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
For testing purposes, you can change this to 
```

on:
  pull_request:
#
```
and change it back to what is currently before merging. This helps you testing it with github actions, but without needing to merge it to master during this testing.

Reply:
yes sure

Reply:
Sure, done.